articles/virtual-machines/workloads/sap/deployment-guide.md

```diff
@@ -986,9 +986,7 @@ To install the Azure Extension for SAP by using Azure CLI:
 1. Install using Azure CLI 2.0
 
    1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
```

Comment:
'Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2]. Make sure to use at least version 2.19.1 or higher'